frontend/src/component/feature/FeatureView/FeatureOverview/StrategyDragTooltip.tsx

```diff
@@ -0,0 +1,101 @@
+import Close from '@mui/icons-material/Close';
+import { Box, Button, IconButton, styled } from '@mui/material';
+import type { FC } from 'react';
+import Joyride, { type TooltipRenderProps } from 'react-joyride';
+
+const StyledTooltip = styled(Box)(({ theme }) => ({
+    display: 'flex',
+    flexDirection: 'column',
+    gap: theme.spacing(1),
+    maxWidth: '300px',
+    background: '#201e42',
```

Comment:
Colors should be used from theme

Reply:
Agreed! Unfortunately, this color isn't in the theme yet (even though it's the same as for the toasts). We should get it in, but that's a different task, I think.